src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/ManageLocationsAndPricesTest.kt

```diff
@@ -27,85 +39,136 @@ internal class ManageLocationsAndPricesTest : IntegrationTest() {
+    isAtPage(Dashboard).navigateToSelectMonthPage()
 
-    isAtPage(Pages.SelectMonthYear).navigateToDashboardFor(currentDate.previousMonth(), year)
+    isAtPage(SelectMonthYear).navigateToDashboardFor(currentDate.previousMonth(), year)
 
-    isAtPage(Pages.Dashboard).navigateToManageJourneyPrice()
+    isAtPage(Dashboard).navigateToManageJourneyPrice()
 
-    isAtPage(Pages.ManageJourneyPriceCatalogue).navigateToFindJourneys()
+    isAtPage(ManageJourneyPriceCatalogue).navigateToFindJourneys()
 
-    isAtPage(Pages.ManageJourneyPrice).findJourneyForPricing("PRISON ONE", "PRISON TWO")
+    isAtPage(ManageJourneyPrice).findJourneyForPricing("PRISON ONE", "PRISON TWO")
 
-    isAtPage(Pages.JourneyResults)
+    isAtPage(JourneyResults)
       .isAtResultsPageForJourney("PRISON ONE", "PRISON TWO")
       .isJourneyRowPresent("PRISON ONE", "PRISON TWO", Money(1000))
       .navigateToUpdatePriceFor("PRISON1", "PRISON2")
 
-    isAtPage(Pages.UpdatePrice)
+    isAtPage(UpdatePrice)
       .isAtPricePageForJourney("PRISON1", "PRISON2")
       .updatePriceForJourney("PRISON1", "PRISON2", Money(2000))
 
-    isAtPage(Pages.JourneyResults)
+    isAtPage(JourneyResults)
       .isAtResultsPageForJourney("PRISON ONE", "PRISON TWO")
       .isPriceUpdatedMessagePresent("PRISON ONE", "PRISON TWO", Money(2000))
       .isJourneyRowPresent("PRISON ONE", "PRISON TWO", Money(2000))
   }
 
   @Test
   @Order(3)
+  fun `price book cannot be changed with journey two years old or more missing price`() {
+    goToPage(Dashboard)
+
+    isAtPage(Login).login()
+
+    isAtPage(ChooseSupplier).choose(Supplier.SERCO)
+
+    isAtPage(Dashboard).navigateToSelectMonthPage()
+
+    isAtPage(SelectMonthYear).navigateToDashboardFor(currentDate.previousMonth(), year.minusYears(2))
+
+    isAtPage(Dashboard).navigateToJourneysForReview()
+
+    isAtPage(JourneysForReview).addPriceForJourney("PRISON1", "PRISON2")
+
+    isAtPage(Dashboard)
+  }
+
+  @Test
+  @Order(4)
+  fun `price book cannot be changed when journey price two years old or more`() {
+    goToPage(Dashboard)
+
+    isAtPage(Login).login()
+
+    isAtPage(ChooseSupplier).choose(Supplier.SERCO)
+
+    isAtPage(Dashboard).navigateToSelectMonthPage()
+
+    isAtPage(SelectMonthYear).navigateToDashboardFor(currentDate.previousMonth(), year.minusYears(2))
+
+    isAtPage(Dashboard).navigateToManageJourneyPrice()
+
+    isAtPage(ManageJourneyPriceCatalogue).navigateToFindJourneys()
+
+    isAtPage(ManageJourneyPrice).findJourneyForPricing("PRISON ONE", "POLICE ONE")
+
+    isAtPage(JourneyResults)
+      .isAtResultsPageForJourney("PRISON ONE", "POLICE ONE")
+      .isJourneyRowPresent("PRISON ONE", "POLICE ONE", Money.valueOf(50.00))
+      .navigateToUpdatePriceFor("PRISON1", "POLICE1")
+
+    isAtPage(UpdatePrice)
+      .isAtPricePageForJourney("PRISON1", "POLICE1")
+      .assertTextIsNotPresent("Update price")
+      .assertTextIsNotPresent("Price exceptions")
+      .assertTextIsPresent("Price history")
```

Comment:
The key here is we should only see the price history (and not the 'update price' or 'price exceptions') when trying to update an existing price.